src/views/RecipeView/Recipe.js

```diff
@@ -0,0 +1,37 @@
+import { Card } from 'antd';
+import { Row, Col } from 'antd';
+
+function Recipe(props) {
+    const listItems = props.ingredientsList.map((ingredient) => <li>{ingredient}</li>);
```

Comment:
When you are mapping to elements you should use `key` property. Here is why: https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js. Also, you can name this variable `ingredientsItemsList` or something that suggests what it is.